bitnami/zookeeper/templates/scripts-configmap.yaml

```diff
@@ -37,9 +37,9 @@ data:
         exit 1
     fi
     {{- else }}
-    elif [[ -f "/certs/client/zookeeper.truststore.jks" ]] && [[ -f "/certs/client/zookeeper.keystore.jks" ]]; then
-        cp "/certs/client/zookeeper.truststore.jks" "/opt/bitnami/zookeeper/config/certs/client/zookeeper.truststore.jks"
-        cp "/certs/client/zookeeper.keystore.jks" "/opt/bitnami/zookeeper/config/certs/client/zookeeper.keystore.jks"
+    elif [[ -f {{ printf "/certs/client/%s" .Values.tls.client.truststoreSecretKey | quote }} ]] && [[ -f {{ printf "/certs/client/%s" .Values.tls.client.keystoreSecretKey | quote }} ]]; then
+        cp {{ printf "/certs/client/%s" .Values.tls.client.truststoreSecretKey | quote }} "/opt/bitnami/zookeeper/config/certs/client/zookeeper.truststore.jks"
+        cp {{ printf "/certs/client/%s" .Values.tls.client.keystoreSecretKey | quote }} "/opt/bitnami/zookeeper/config/certs/client/zookeeper.keystore.jks"
```

Comment:
This is nice, but I would use a common approach found in other charts, which is to define a new helper function that will return the right name:
https://github.com/bitnami/charts/blob/c118f228f5c932f789a31782d992986b64508f70/bitnami/redis/templates/_helpers.tpl#L172-L181

Reply:
I checked to define a new helper function for this, but I do not find the point of a helper function here. It will return always the same value? Moreover, I will need 4 helper functions. I think it will just complexify. I would skip this one. What do you think?

Reply:
> but I do not find the point of a helper function here. It will return always the same value?

The thing here is that the keystore file name (that is, the key in the secret) might not be the default one expected by the chart. When an `existingSecret` is not provided, the keystore is named `zookeeper.truststore.jks`. However, if you use Cert-Manager, it will be `truststore.jks`. Maybe I misunderstood or made a wrong statement, but you can see that they might have different values.

By the way (maybe the confusion comes from this), the default value I expect for `existingSecretKeystoreKey` is `""` and NOT `"zookeeper.truststore.jks"` (which is the chart's default value). To me, this parameter conceptually exclusively models the name of the KeystoreKey (a.k.a. the keystore file) in the `existingSecret`.

Reply:
> but I do not find the point of a helper function here. It will return always the same value?

The thing here is that the keystore file name (that is, the key in the secret) might not be the default one expected by the chart. When an `existingSecret` is not provided, the keystore is named `zookeeper.truststore.jks`. However, if you use Cert-Manager, it will be `truststore.jks`. Maybe I misunderstood or made a wrong statement, but you can see that they might have different values.

By the way (maybe the confusion comes from this), the default value I expect for `existingSecretKeystoreKey` is `""` and NOT `"zookeeper.truststore.jks"` (which is the chart's default value). To me, this parameter conceptually exclusively models the name of the KeystoreKey (a.k.a. the keystore file) in the `existingSecret`.

Reply:
You were right about the confusion :D  
I updated both the default values and the functions helper!